packages/core/src/html/NodeProcessor.js

```diff
@@ -264,9 +265,7 @@ class NodeProcessor {
     if (linkProcessor.hasTagLink(node)) {
       linkProcessor.convertRelativeLinks(node, context.cwf, this.config.rootPath, this.config.baseUrl);
       linkProcessor.convertMdExtToHtmlExt(node);
-      if (this.config.intrasiteLinkValidation.enabled) {
-        linkProcessor.validateIntraLink(node, context.cwf, this.config);
-      }
+      this.siteLinkManager.collectIntraLink(node, context.cwf);
```

Comment:
This will collect intralinks even when validation is not enabled. Should we add a check here whether it is the case?